### PR TITLE
Uar-266 registration mentioned in update bug fix

### PR DESCRIPTION
**Pull request by garywco**
### JIRA link

https://companieshouse.atlassian.net/jira/software/c/projects/UAR/boards/481?modal=detail&selectedIssue=UAR-266

### Change description

Fix to remove "register" from sign out text when save and resume is active. The text will now be replaced and say "update" instead.

### Work checklist

- [ ] Tests added where applicable
- [ ] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: test/controllers/sign.out.controller.spec.ts [hunk 1]
```diff
@@ -51,6 +71,7 @@ describe("Sign Out controller", () => {
     });
 
     test("catch error when rendering the page", async () => {
+      mockIsActiveFeature.mockReturnValueOnce(true);
```
Comment (AndreKramer7654) [inline test/controllers/sign.out.controller.spec.ts:74]: Mock on line 74 not strictly needed.

File: test/controllers/sign.out.controller.spec.ts [hunk 2]
```diff
@@ -17,6 +22,8 @@ import {
 } from "../__mocks__/text.mock";
 
 import { createAndLogErrorRequest, logger } from '../../src/utils/logger';
+const mockIsActiveFeature = isActiveFeature as jest.Mock;
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
```
Comment (AndreKramer7654) [inline test/controllers/sign.out.controller.spec.ts:26]: Please separate this into two lines.

File: test/controllers/update/sign.out.controller.spec.ts [hunk 3]
```diff
@@ -19,10 +20,12 @@ import {
 
 import { serviceAvailabilityMiddleware } from "../../../src/middleware/service.availability.middleware";
 import { NextFunction, Request, Response } from "express";
+import { isActiveFeature } from "../../../src/utils/feature.flag";
 
+const mockIsActiveFeature = isActiveFeature as jest.Mock;
+const mockServiceAvailabilityMiddleware = serviceAvailabilityMiddleware as jest.Mock;mockServiceAvailabilityMiddleware.mockImplementation((req: Request, res: Response, next: NextFunction) => next() );
```
Comment (AndreKramer7654) [inline test/controllers/update/sign.out.controller.spec.ts:26]: Please separate into two lines.

File: test/controllers/update/sign.out.controller.spec.ts [hunk 4]
```diff
@@ -40,6 +44,16 @@ describe("SIGN OUT controller", () => {
       expect(resp.text).toContain(`${config.UPDATE_AN_OVERSEAS_ENTITY_URL}${config.SECURE_UPDATE_FILTER_PAGE}`);
     });
 
+    test(`renders the ${config.SIGN_OUT_PAGE} page, when FEATURE_FLAG_ENABLE_SAVE_AND_RESUME is not active,  with ${config.SECURE_UPDATE_FILTER_PAGE} as back link`, async () => {
```
Comment (AndreKramer7654) [inline test/controllers/update/sign.out.controller.spec.ts:47]: again FEATURE_FLAG_ENABLE_UPDATE_SAVE_AND_RESUME in title?

File: views/includes/page-templates/sign-out.html [hunk 5]
```diff
@@ -1,73 +0,0 @@
-{% extends "layout.html" %}
```
Comment (AndreKramer7654) [inline views/includes/page-templates/sign-out.html:1]: This file should be deleted not just made empty

Comment (AndreKramer7654) [reply views/includes/page-templates/sign-out.html:1]: and that was done before the bug fix?

**Review by AndreKramer7654 — APPROVED** [review]

**Review by cdrainCH — APPROVED** [review]

**Pull request merged**